tests/e2e/fixtures/metrics/custom-queries.yaml

```diff
@@ -58,7 +58,7 @@ data:
             description: "Disk space used by the database"
 
   additional-queries: |
-    pg_replication_slots:
+    pg_replication_slots_status:
```

Comment:
Can we use a name that will be always unique, like `e2e_tests_replication_slots_status`?

Reply:
Fixed.